pkg/util/log/log.go

```diff
@@ -30,7 +30,8 @@ const (
 
 // SetLog sets logrus logger (format, file, level).
 func SetLog(logLevel, logFileName string) (*os.File, error) {
-	var f *os.File
+	var logfile *os.File
```

Comment:
Something seems wrong here. Was `f` shadowed in the inner scope?

Reply:
no. It was a bad fix on my part in #269, where I had missed the setting of `f` inside the following block. Since I added `logfile` *and* `:=` in the inner block, `f` was never set to the intended return value in `os.OpenFile()`.
Pushing a fix...